lib/api/client-commands/frame.js

```diff
@@ -39,14 +40,18 @@ const findElement = function(selector) {
   });
 };
 
-module.exports = class Session extends ProtocolAction {
+class Frame extends ClientCommand {
   async command(frameId, callback) {
     if (arguments.length === 1 && typeof frameId === 'function') {
       callback = frameId;
 
       return this.transportActions.switchToFrame(callback);
     }
 
+    if (frameId instanceof Element) {
+      frameId = frameId.selector;
+    }
+
```

Comment:
This change is not very easy to understand just by looking at it.

Can you add a very brief explanation above this change (in the form of a comment) explaining in what case `frameId` can be an instance of `Element` and why we are setting `frameId` to its selector property (and benefits & caveats of doing so)?

Reply:
Or just write it here first so that we can discuss over it.

Reply:
Check if frameId is an HTML element in DOM and convert it into its CSS selector representation.